test/Riok.Mapperly.IntegrationTests/_snapshots/NET_48/StaticMapperTest.SnapshotGeneratedSource.verified.cs

```diff
@@ -319,7 +323,18 @@ public static partial class StaticTestMapper
             return target;
         }
 
-        private static global::System.Collections.Generic.Stack<int> MapToStack(global::System.Collections.Generic.Stack<string> source)
+        private static global::System.Collections.Generic.Stack<int> MapToStack(global::System.Collections.Generic.IEnumerable<string> source)
+        {
+            var target = new global::System.Collections.Generic.Stack<int>();
+            foreach (var item in source)
```

Comment:
Shouldn't this be the point where the type tests should be emitted? As mentioned in another comment, I'm also ok with removing the `EnsureCapacityIsType` but then this integration test property could also be removed.